samples/hello-cube.html

```diff
@@ -0,0 +1,309 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta name="viewport" content="width=600">
+<title>Web GPU Cube demo</title>
+<script src="gl-matrix-min.js"></script>
+</head>
+<body>
+<canvas height=1000 width=1000></canvas>
+<script>
+const shader = `
```

Comment:
As before, before merge, need a big scary warning explaining the state of shader things: This is a preview of WHLSL, but the group hasn't decided what shading language will be taken in by the API.

Reply:
Updated!

Reply:
Thanks! LGTM

Reply:
(on this fix. sorry, haven't reviewed the rest yet)